src/test/java/edu/cornell/kfs/module/purap/document/service/impl/CuCreditMemoServiceImplTest.java

```diff
@@ -62,28 +60,26 @@
     private PurapService purapService;
     private VendorService vendorService;
     private CuVendorCreditMemoDocument creditMemoDocument;
-    @Mock
     private static Person mo14Person;
-    @Mock
     private static UserSession mo14Session;
 
     @Before
 	public void setUp() throws Exception {
-        accountsPayableService = EasyMock.createNiceMock(AccountsPayableService.class);
-        EasyMock.expect(accountsPayableService.getExpiredOrClosedAccountList(creditMemoDocument)).andReturn(new HashMap<>());
-        EasyMock.replay(accountsPayableService);
-
-        dataDictionaryService = EasyMock.createNiceMock(DataDictionaryService.class);
-        EasyMock.expect(dataDictionaryService.getAttributeMaxLength(DocumentHeader.class, KRADPropertyConstants.DOCUMENT_DESCRIPTION)).andReturn(200);
-        EasyMock.replay(dataDictionaryService);
-
+        accountsPayableService = Mockito.mock(AccountsPayableService.class);
+        Mockito.when(accountsPayableService.getExpiredOrClosedAccountList(creditMemoDocument)).thenReturn(new HashMap<>());
+        
+        dataDictionaryService = Mockito.mock(DataDictionaryService.class);
+        Mockito.when(dataDictionaryService.getAttributeMaxLength(DocumentHeader.class, KRADPropertyConstants.DOCUMENT_DESCRIPTION)).thenReturn(200);
+        
         dateTimeService = new DateTimeServiceImpl();
-        documentService = new MockDocumentServiceImpl();
-        noteService = EasyMock.createMock(NoteService.class);
-        purapService = EasyMock.createMock(PurapService.class);
+        documentService = buildMockDocumentService();
+        noteService = Mockito.mock(NoteService.class);
+        purapService = Mockito.mock(PurapService.class);
         vendorService = new MockVendorServiceImpl();
 
-        creditMemoServiceImpl = PowerMock.createPartialMock(CuCreditMemoServiceImplTest.TestCuCreditMemoServiceImpl.class, "reIndexDocument", "getCreditMemoDocumentById");
+        
+        creditMemoServiceImpl = PowerMockito.spy(new TestCuCreditMemoServiceImpl());
+        PowerMockito.doNothing().when(creditMemoServiceImpl, "reIndexDocument", Mockito.any());
```

Comment:
I see that "getCreditMemoDocumentById" is no longer being mocked. Did it not need mocking anymore (possibly due to "nice" mocking functionality in Mockito)?

Reply:
I was looking back at this, it looks like getCreditMemoDocumentById doesn't need to be mocked any longer to meet the needs of this test.

Reply:
Ok, thanks for checking.